src/ucp/core/ucp_worker.c

```diff
@@ -1327,7 +1327,12 @@ static ucs_status_t ucp_worker_add_resource_cms(ucp_worker_h worker)
         worker->cms[i++].cmpt_idx = cmpt_index;
     }
 
-    status = UCS_OK;
+    worker->timer.id   = 0;
+    worker->timer.tick = ucs_time_from_sec(context->config.ext.cm_close_timeout);
```

Comment:
seems the variable timer.tick is not needed, can use context->config.ext.cm_close_timeout when needed to set close request timeout field